samples/client/petstore/java/jersey2-java6/docs/MixedPropertiesAndAdditionalPropertiesClass.md

```diff
@@ -4,7 +4,7 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**uuid** | **String** |  |  [optional]
+**uuid** | [**UUID**](UUID.md) |  |  [optional]
```

Comment:
That looks wrong. (The `format: uuid` was in the YAML from the beginning. Possibly #4739 changing to use java.util.UUID instead of String added the link here?

Related: #5026. #5027 might possibly fix this too.